### PR TITLE
Renaming in changelog and FAQs

### PR DESCRIPTION
**Pull request by adfonseca**
## Description
Renaming of products Mercado Pago in changelog and FAQs

### Issue involved
Fixes #<issue>

### Checklist:
Before sending your contribution, please specify the following: 

- [ ] This request modifies code snippets. 
- [ ] The contribution aligns with the information stated in the repository wiki.

- [ ] Contribution incorporates an article not included until now, which must be added to indexes as a new section. 
- [ ] Contribution includes new features. 
- [ ] New features were communicated in changelog.
- [ ] Requires translations.

### CODE REVIEW

File: guides/changelog/current-year.es.md [hunk 1]
```diff
@@ -213,15 +213,15 @@ Disponibilizamos una nueva versión de SDK .NET.
 
 > CHANGELOG
 >
-> Actualización de documentación de API
+> Actualización de documentación de Checkout API
```
Comment (Juanii95) [inline guides/changelog/current-year.es.md:216]: Para MLB es Checkout Transparente

File: guides/changelog/current-year.es.md [hunk 2]
```diff
@@ -213,15 +213,15 @@ Disponibilizamos una nueva versión de SDK .NET.
...
 >
 > NEW_DOCUMENTATION: NUEVA DOCUMENTACIÓN
 >
-> PRODUCT: API
+> PRODUCT: CHECKOUT API
```
Comment (Juanii95) [inline guides/changelog/current-year.es.md:220]: Para MLB es Checkout Transparente

File: guides/changelog/current-year.es.md [hunk 3]
```diff
@@ -213,15 +213,15 @@ Disponibilizamos una nueva versión de SDK .NET.
...
 
 Renovamos la documentación para usar nuestras APIs de pagos. Agregamos un ejemplo básico del formulario en el frontend y sumamos diagramas de funcionamiento para ayudarte a la hora de integrar.
 
-[Ir a documentación de API](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/payments/api/introduction/)
+[Ir a documentación de Checkout API](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/payments/api/introduction/)
```
Comment (Juanii95) [inline guides/changelog/current-year.es.md:224]: Para MLB es Checkout Transparente

File: guides/changelog/current-year.pt.md [hunk 4]
```diff
@@ -212,15 +212,15 @@ Lançamos uma nova versão do SDK .NET.
 
 > CHANGELOG
 >
-> Atualização de documentação de API
+> Atualização de documentação de Checkout Transparente
```
Comment (Juanii95) [inline guides/changelog/current-year.pt.md:215]: Para MLA es Checkout API

File: guides/changelog/current-year.pt.md [hunk 5]
```diff
@@ -212,15 +212,15 @@ Lançamos uma nova versão do SDK .NET.
...
 >
 > NEW_DOCUMENTATION: NOVA DOCUMENTAÇÃO
 >
-> PRODUCT: API
+> PRODUCT: CHECKOUT TRANSPARENTE
```
Comment (Juanii95) [inline guides/changelog/current-year.pt.md:219]: Para MLA es Checkout API

File: guides/changelog/current-year.pt.md [hunk 6]
```diff
@@ -212,15 +212,15 @@ Lançamos uma nova versão do SDK .NET.
...
 
 Renovamos a documentação para usar nossas APIs de pagamento. Adicionamos um exemplo básico do formulário no frontend e também diagramas de funcionamento para te ajudar na hora de integrar.
 
-[Ir para a documentação de API](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/payments/api/introduction/)
+[Ir para a documentação de Checkout Transparente](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/payments/api/introduction/)
```
Comment (Juanii95) [inline guides/changelog/current-year.pt.md:223]: Para MLA es Checkout API

**Comment by Juanii95** [comment]
Por último, en el changelog `.en` falta modifica API por Checkout API

File: guides/changelog/current-year.en.md [hunk 7]
```diff
@@ -210,17 +210,35 @@ We have released a new version of SDK .NET.
...
+
+----[mlb]----
+> CHANGELOG
+>
+> Actualización de documentación de Checkout Transparente
+>
+> NEW_DOCUMENTATION: NUEVA DOCUMENTACIÓN
+>
+> PRODUCT: CHECKOUT TRANSPARENTE
+------------
```
Comment (pamalv7) [inline guides/changelog/current-year.en.md:231]: @adfonseca no es necesario esta parte, se llaman igual en all sites (y quedó en español)

File: guides/faqs/payments.pt.md [hunk 8]
```diff
@@ -64,7 +64,7 @@ No retorno da chamada haverá o parâmetro `transaction_details` que conterá o
 
 Dependendo do seu tipo de checkout e integração o processo pode ser diferente.
 
-### &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Checkout Mercado Pago
+### &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Checkout Pro
```
Comment (pamalv7) [inline guides/faqs/payments.pt.md:67]: @adfonseca  No falta también el título "APIs Mercado Pago"?

File: guides/changelog/current-year.en.md [hunk 9]
```diff
@@ -118,15 +118,15 @@ We have released a new version of SDK .NET 1.8.0.
 
 > CHANGELOG
 >
-> New version for Button and payment link
+> New version for Payment link
```
Comment (pamalv7) [inline guides/changelog/current-year.en.md:121]:
```suggestion
> New version for Payment Link
```

File: guides/changelog/current-year.en.md [hunk 10]
```diff
@@ -118,15 +118,15 @@ We have released a new version of SDK .NET 1.8.0.
...
 >
 > NEW_VERSION: NEW VERSION
 >
-> PRODUCT: BUTTON AND PAYMENT LINK
+> PRODUCT: PAYMENT LINK
 
-We have lauched a new version for Button and payment link, totally renewed. We have designed and implemented a new experience when creating payment links to share online by chat, e-mail or social media.
+We have lauched a new version for Payment link, totally renewed. We have designed and implemented a new experience when creating payment links to share online by chat, e-mail or social media.
```
Comment (pamalv7) [inline guides/changelog/current-year.en.md:127]:
```suggestion
We have lauched a new version for Payment Link, totally renewed. We have designed and implemented a new experience when creating payment links to share online by chat, e-mail or social media.
```

File: guides/changelog/current-year.en.md [hunk 11]
```diff
@@ -118,15 +118,15 @@ We have released a new version of SDK .NET 1.8.0.
...
 
-[Go to Button and payment link documentation](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/payments/button/intro-button/)
+[Go to Payment link documentation](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/payments/button/intro-button/)
```
Comment (pamalv7) [inline guides/changelog/current-year.en.md:129]:
```suggestion
[Go to Payment Link documentation](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/payments/button/intro-button/)
```

File: guides/faqs/integrations.en.md [hunk 12]
```diff
@@ -50,15 +50,21 @@ It's the simplified version of the Mercado Pago Checkout with a renewed user exp
 
 [Integrate Web Tokenize](https://www.mercadopago.com.ar/developers/en/guides/payments/web-tokenize-checkout/introduction)
 
-### API
+----[mlm, mla, mlc, mco, mpe, mlu]----
+### Checkout API
+------------
+
+----[mlb]----
+### Checkout Transparente
```
Comment (pamalv7) [inline guides/faqs/integrations.en.md:58]: En inglés es siempre **Checkout API**

File: guides/faqs/integrations.es.md [hunk 13]
```diff
@@ -43,15 +43,21 @@ Es la versión simplificada del Checkout de Mercado Pago con una experiencia de
...
+------------
 
 Esta forma de integrarse permite conectarse directamente con nuestras APIs.
 
 - **Convertite en el dueño de tu experiencia.** Te proporcionamos absoluta libertad para personalizar la experiencia de tus compradores y que nunca salgan de tu sitio.  
 
 - **Tu equipo técnico tiene el control total.** Van a poder manejar todos los procesos de manipulación de datos sensibles en el front-end donde se produce la solicitud de pago.
 
-[Integrar por API](https://www.mercadopago.com.ar/developers/es/guides/payments/api/introduction)
+[Integrar por Checkout API](https://www.mercadopago.com.ar/developers/es/guides/payments/api/introduction)
```
Comment (pamalv7) [inline guides/faqs/integrations.es.md:60]: Checkout Transparente para MLB

File: guides/faqs/integrations.pt.md [hunk 14]
```diff
@@ -49,15 +49,21 @@ Diferente do Checkout Mercado Pago, facilitamos a criação de uma experiência
...
+------------
 
 Esta forma de integração permite que você se conecte diretamente com as nossas APIs.
 
 - **Torne-se dono da sua experiência.** Proporcionamos liberdade absoluta para você personalizar as experiências dos seus compradores e que nunca saiam do seu site.  
 
 - **Sua equipe técnica tem total controle.** Eles poderão lidar com todos os processos de manipulação de dados sensíveis no front-end, onde a solicitação de pagamento ocorre.
 
-[Integrar por API](https://www.mercadopago.com.br/developers/pt/guides/payments/api/introduction)
+[Integrar por Checkout Transparente](https://www.mercadopago.com.br/developers/pt/guides/payments/api/introduction)
```
Comment (pamalv7) [inline guides/faqs/integrations.pt.md:66]: **Checkout API** para MLA

File: guides/changelog/current-year.en.md [hunk 15]
```diff
@@ -118,7 +118,7 @@ We have released a new version of SDK .NET 1.8.0.
 
 > CHANGELOG
 >
-> New version for Payment link
+> New version for Payment Link
```
Comment (adfonseca) [inline guides/changelog/current-year.en.md:121]:
```suggestion
> New version for Payment link
```

File: guides/faqs/integrations.es.md [hunk 16]
```diff
@@ -375,15 +429,27 @@ Es la versión simplificada del Checkout de Mercado Pago con una experiencia de
...
 
 Esta forma de integrarse permite conectarse directamente con nuestras APIs.
 
 - **Convertite en el dueño de tu experiencia.** Te proporcionamos absoluta libertad para personalizar la experiencias de tus compradores y que nunca salgan de tu sitio.  
 
 - **Tu equipo técnico tiene el control total.** Van a poder manejar todos los procesos de manipulación de datos sensibles en el front-end donde se produce la solicitud de pago.
 
-[Integrar por API](https://www.mercadopago.com.ar/developers/es/guides/payments/api/introduction)
+----[mlm, mla, mlc, mco, mpe, mlu]----
+[Integrar por Checkout API](https://www.mercadopago.com.ar/developers/es/guides/payments/api/introduction)
```
Comment (pamalv7) [inline guides/faqs/integrations.es.md:447]:
```suggestion
[Integrar Checkout API](https://www.mercadopago.com.ar/developers/es/guides/payments/api/introduction)
```

File: guides/faqs/integrations.es.md [hunk 17]
```diff
@@ -375,15 +429,27 @@ Es la versión simplificada del Checkout de Mercado Pago con una experiencia de
...
+------------
+
+----[mlb]----
+[Integrar por Checkout Transparente](https://www.mercadopago.com.ar/developers/es/guides/payments/api/introduction)
```
Comment (pamalv7) [inline guides/faqs/integrations.es.md:451]:
```suggestion
[Integrar Checkout Transparente](https://www.mercadopago.com.ar/developers/es/guides/payments/api/introduction)
```

File: guides/faqs/integrations.pt.md [hunk 18]
```diff
@@ -49,15 +49,27 @@ Diferente do Checkout Mercado Pago, facilitamos a criação de uma experiência
...
 
 Esta forma de integração permite que você se conecte diretamente com as nossas APIs.
 
 - **Torne-se dono da sua experiência.** Proporcionamos liberdade absoluta para você personalizar as experiências dos seus compradores e que nunca saiam do seu site.  
 
 - **Sua equipe técnica tem total controle.** Eles poderão lidar com todos os processos de manipulação de dados sensíveis no front-end, onde a solicitação de pagamento ocorre.
 
-[Integrar por API](https://www.mercadopago.com.br/developers/pt/guides/payments/api/introduction)
+----[mlm, mla, mlc, mco, mpe, mlu]----
+[Integrar por Checkout API](https://www.mercadopago.com.br/developers/pt/guides/payments/api/introduction)
```
Comment (pamalv7) [inline guides/faqs/integrations.pt.md:67]:
```suggestion
[Integrar Checkout API](https://www.mercadopago.com.br/developers/pt/guides/payments/api/introduction)
```

File: guides/faqs/integrations.pt.md [hunk 19]
```diff
@@ -49,15 +49,27 @@ Diferente do Checkout Mercado Pago, facilitamos a criação de uma experiência
...
+------------
+
+----[mlb]----
+[Integrar por Checkout Transparente](https://www.mercadopago.com.br/developers/pt/guides/payments/api/introduction)
```
Comment (pamalv7) [inline guides/faqs/integrations.pt.md:71]:
```suggestion
[Integrar Checkout Transparente](https://www.mercadopago.com.br/developers/pt/guides/payments/api/introduction)
```

File: guides/faqs/integrations.es.md [hunk 20]
```diff
@@ -113,15 +125,27 @@ Es la versión simplificada del Checkout de Mercado Pago con una experiencia de
...
 
 Esta forma de integrarse permite conectarse directamente con nuestras APIs.
 
 - **Convertite en el dueño de tu experiencia.** Te damos absoluta libertad para personalizar la experiencias de tus compradores y que nunca salgan de tu sitio.  
 
 - **Tu equipo técnico tiene el control total.** Van a poder manejar todos los procesos de manipulación de datos sensibles en el front-end donde se produce la solicitud de pago.
 
-[Integrar por API](https://www.mercadopago.com.ar/developers/es/guides/payments/api/introduction)
+----[mlm, mla, mlc, mco, mpe, mlu]----
+[Integrar por Checkout API](https://www.mercadopago.com.ar/developers/es/guides/payments/api/introduction)
```
Comment (pamalv7) [inline guides/faqs/integrations.es.md:143]:
```suggestion
[Integrar Checkout API](https://www.mercadopago.com.ar/developers/es/guides/payments/api/introduction)
```

File: guides/faqs/integrations.es.md [hunk 21]
```diff
@@ -113,15 +125,27 @@ Es la versión simplificada del Checkout de Mercado Pago con una experiencia de
...
+------------
+
+----[mlb]----
+[Integrar por Checkout Transparente](https://www.mercadopago.com.ar/developers/es/guides/payments/api/introduction)
```
Comment (pamalv7) [inline guides/faqs/integrations.es.md:147]:
```suggestion
[Integrar Checkout Transparente](https://www.mercadopago.com.ar/developers/es/guides/payments/api/introduction)
```

File: guides/faqs/integrations.en.md [hunk 22]
```diff
@@ -50,15 +50,15 @@ It's the simplified version of the Mercado Pago Checkout with a renewed user exp
...
+### Checkout API
 
 This way of integrating allows you to connect directly with our APIs.
 
 - **Become the owner of your experience.** We give you absolute freedom to customize the experiences of your buyers and never leave your site.  
 
 - **Your technical team has total control.** They will be able to handle all sensitive data manipulation processes in the front-end where the payment request occurs.
 
-[Integrate by API](guides/payments/api/introduction)
+[Integrate by Checkout API](guides/payments/api/introduction)
```
Comment (pamalv7) [inline guides/faqs/integrations.en.md:61]:
```suggestion
[Integrate Checkout API](guides/payments/api/introduction)
```

**Pull request merged**